SQL/paradise_schema.sql

```diff
@@ -250,6 +250,7 @@ CREATE TABLE `player` (
   `nanoui_fancy` smallint(4) DEFAULT '1',
   `show_ghostitem_attack` smallint(4) DEFAULT '1',
   `lastchangelog` varchar(32) NOT NULL DEFAULT '0',
+  `exp` mediumtext,
```

Comment:
Like "be_role" in the same table, "exp" is a comma-separated list of values. It tracks the number of minutes that the player has spent in each category for which stats are tracked. Here's an example value from my test database: "Living=610&Crew=610&Special=0&Ghost=30&Exempt=0". Translated, this means I've spent 10h10m as a living mob, 10h10m as a crew member, 30 minutes as a ghost, no time as a special role, and I'm not flagged Exempt. Records like that test record will be set for each active player, and it is 'mediumtext' because that's the type used for similar storage fields like 'be_role'.